lib/owl_plot.ml

```diff
@@ -1593,14 +1593,24 @@ let scatterhist = None
 
 (* other plots *)
 
+let _rotate270_ x =
+  let w, h = Owl_dense_matrix.D.shape x in
```

Comment:
I will implement a `flip` function in Matrix module later today. Before that, a better way is to use `slicing` function. E.g.,

```ocaml
Mat.slice [[-1;0];[-1;0]] x
```

Less code and way faster :)